### PR TITLE
fix error object marshalling

### PR DESCRIPTION
**Pull request by javaes**

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -19,6 +22,10 @@
 from figo.models import Transaction
 from figo.models import User
 
+from tests.test_writing_methods import CLIENT_ERROR
+
+HTTP_NOT_ACCEPTABLE = 406
```
Comment (denvercoder9) [inline tests/test_models.py]: Wouldn't it be better to have something like a constants module also for the SDK?

File: tests/test_writing_methods.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 
 CREDENTIALS = ["figo", "figo"]
 BANK_CODE = "90090042"
+CLIENT_ERROR = 1000
```
Comment (denvercoder9) [inline tests/test_writing_methods.py]: Same as above...

File: tests/test_models.py (hunk 1 above)
Comment (moritzpein) [reply tests/test_models.py]: `httplib.NOT_ACCEPTABLE`

File: tests/test_writing_methods.py (hunk 2 above)
Comment (moritzpein) [reply tests/test_writing_methods.py]: Above I would use the stdlib (`httplib`) provided constant, these here should move into a constant-module, but maybe not in this PR.

**Pull request merged**